pkg/clients/resourcerecordset/resourcerecordset.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 The Crossplane Authors.
```

Comment:
```suggestion
Copyright 2020 The Crossplane Authors.
```

The date here is about when the file is created.

Reply:
The proper date and license has been added to all the files.